index.js

```diff
@@ -149,10 +155,15 @@ SQLdown.prototype._get = function (key, options, cb) {
     if (!res.length) {
       return cb(new Error('NotFound'));
     }
+
     try {
       var value = JSON.parse(res[0].value);
       if (asBuffer) {
-        value = new Buffer(value);
+        if(value.type === 'Buffer'){
```

Comment:
so i don't think you can't just stringy values into buffers. for example if you have a buffer like this
```
var b = new Buffer([4, 5, 6]);
b.toString() //= ""
```
now we could do this if we base64 encodeded it or something.